_client/src/UiManager.cpp

```diff
@@ -19,11 +22,10 @@ UiManager::UiManager(ApiManager *apiClient, QObject *parent) noexcept
     // Initialize all necessary elements
     init();
     setupApiConnections();
-    setupDialogsConnections();
     setupMVVMConnections();
 
     // Before initialization
-    m_mainWindow->hide();
+    m_mainWindow->setEnabled(false);
```

Comment:
If you set this, then you need to implement logic of `setEnable(true)` after `LoginDialog` gets `LoginSuccess`

Reply:
restore, this'll be another task